configs/image_restorers/srcnn/metafile.yml

```diff
@@ -18,6 +19,7 @@ Models:
       DIV2K:
         PSNR: 27.746
         SSIM: 0.7854
+      GPU Info: 1.0
```

Comment:
Parse error, I will solve it by updating ``.dev_scripts/github/update_model_index.py``